csharp/Managed/Events/ListEventArgs.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using LibPDBinding.Managed.Data;
+using LibPDBinding.Managed.Utils;
+
+namespace LibPDBinding.Managed.Events
+{
+	public class ListEventArgs : EventArgs
+	{
+		public string Receiver { get; private set; }
+
+		public IEnumerable<IAtom> List { get; private set; }
```

Comment:
Alternative could be `IAtom[]` or `IEnumerable<object>` or `object[]`. 

- 'IEnumerable` instead of `[]`: most C# API use it and is more like Pd lists look like from coding
- `IAtom` instead of `object`: makes it clear what kind of items can be in the list.